### PR TITLE
Updated few fixed on deployment guide for TKO-on-vSphere

### PR DESCRIPTION
**Pull request by samiksha-j**
1. Updated the internal link to deploy NSX ALB secondary servers. 
2. Added AKODeploymentConfig warning (we get while deploying the managment cluster in tkg 1.4.x) as a note.
3. Updated the Yaml file for AKO deployment config as it was incorrect in format. 
4. Updated Contour-data-values.yaml file due to similar reason as in point number 3.

### CODE REVIEW

**Comment by craigtracey** [comment]
Thanks for the PR @samiksha-j. As this repo requires that commits are signed off, we will need to address this before we can merge these changes.  In this particular case, this will likely amount to squashing the 4 commits into 1 and then signing that new single commit.i

An easy way to achieve this would be:
```
git remote add upstream git@github.com:vmware-tanzu-labs/tanzu-validated-solutions
git fetch upstream
git reset HEAD~4
git stash
git rebase upstream/main
git stash pop
```
As this file has been updated since your changes, you will need to resolve the conflict in `src/deployment-guides/tko-on-vsphere.md`

Once conflict is fixed:
```
git commit -a -v -s
<enter your commit message>
git push origin main
```

Hope this helps...please reach out if you need further assistance.

**Comment by samiksha-j** [comment]
Thank you @craigtracey for calling out the issues and guiding me with the process to fix it. 

I have made the recommended changes and pushed it to this main repository. 

Feel free to let me know if anything else needs to done.

File: src/deployment-guides/tko-on-vsphere.md [hunk 1]
```diff
@@ -645,6 +645,8 @@ To get the status of TKG Management cluster execute the following command:
     ![](img/tko-on-vsphere/image51.png)
 
 The TKG management cluster is successfully deployed and now you can created Shared Service and workload clusters  
+	
+> **_NOTE:_**  In version 1.4.x, you can receive a warning prompt for AKODeploymentConfig while deploying the management cluster. You can ignore the warning, for now, it will not create any issue.
```
Comment (craigtracey) [inline src/deployment-guides/tko-on-vsphere.md:649]: We should remove this as this deployment guide is targeting 1.5.x, and 1.4.x has already been shipped.

Comment (manishjha86) [reply src/deployment-guides/tko-on-vsphere.md:649]: This is already fixed in TKO 1.5 and as Craig said, we will not modify TKO 1.4 doc.

**Comment by manishjha86** [comment]
@samiksha-j let us know if you still need any assistance on this issue. If my last comment makes sense, please close this issue as no action is needed.

**Comment by samiksha-j** [comment]
sure @manishjha86  we can close this one.

**Pull request closed without merging**